webapp-django/crashstats/crashstats/templates/crashstats/crashes_per_user.html

```diff
@@ -0,0 +1,275 @@
+{% extends "crashstats_base.html" %}
+{% block site_css %}
+  {{ super() }}
+  {% compress css %}
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.css') }}" media="screen" />
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.structure.css') }}" media="screen" />
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.theme.css') }}" media="screen" />
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/metricsgraphics.css') }}">
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/metricsgraphics_custom.css') }}">
+  {% endcompress %}
+  {% compress css %}
+      <link rel="stylesheet" type="text/less" href="{{ static('crashstats/css/crashes_per_user.less') }}">
+  {% endcompress %}
+{% endblock %}
+
+{% block page_title %}
+Crashes per User for {{ product }}
+{% endblock %}
+
+{% block content %}
+<div id="mainbody">
+
+<div class="page-heading">
+    <p style="float: right">
```

Comment:
I've moved it to the `.less` file now. 
It's a strange issue actually. I've studied the subject some and listened to a long podcast about it. 
There's a school of thought that the only thing you do with global CSS is things like `reset.css` and some odd exceptions. Everything else you do inline styles. Actually, in an era of tools like React you don't do `<div id="body" style="font-family:...">` because you don't first create the HTML at all. Instead you just write the styles directly into th DOM elements. And instead of setting `backgroundColor="red"` you use `backgroundColor=generalBackgroundColorTheme1` so it' variables and not values. 
The advantage is that styling becomes tied to the code that actually needs it. 
However, I'm not sold on it. It has advantages and disadvantages.